lib/typescript/typescript_transpiler.js

```diff
@@ -0,0 +1,33 @@
+import ts from 'typescript';
+import fs from 'fs';
+import path from 'path';
+
+const TYPESCRIPT_COMPILER_OPTIONS = {
+    compilerOptions: {
+        module: ts.ModuleKind.ESNext,
+        target: ts.ScriptTarget.ES2020,
+        jsx: ts.JsxEmit.Preserve,
+    },
+};
+
+export class TypescriptTranspiler {
+    static async generateTemporalJavascriptFromTypescript(filePath) {
+        const tsCode = fs.readFileSync(filePath, 'utf8');
```

Comment:
Is it possible to have other encodings? Will the transpiler fail in that case?

Reply:
The vast majority of modern code editors and development environments default to UTF-8 for source code. It's the common standard, so we can add it to the documentation to make sure users know it.